osu.Game.Rulesets.Osu/Difficulty/Skills/Speed.cs

```diff
@@ -20,20 +22,119 @@ public class Speed : Skill
         private const double pi_over_4 = Math.PI / 4;
         private const double pi_over_2 = Math.PI / 2;
 
+        /// <summary>
+        /// streamaimconst is derived from the last stream of Megalovania [Excors] 9* map (Unit:pixel/ms)
+        /// </summary>
+        private const double streamaimconst = 2.42;
+
+        /// <summary>
+        /// splitter value for jumpDistances2 list
+        /// </summary>
+        private int sdsplitcounter;
+
+        /// <summary>
+        /// splitter value for jumpDistances list
+        /// </summary>
+        private int sdsplitcounter2;
+
+        /// <summary>
+        /// Final result of SD calculation and scaling added at the result of StrainValueOf(DifficultyHitObject current)
+        /// </summary>
+        private double sdstrainmult;
```

Comment:
I don't understand what this is trying to say. It's still inscrutable.

Reply:
```
return ((1 + (speedBonus - 1) * 0.75) * angleBonus * (0.95 + speedBonus * Math.Pow(distance / single_spacing_threshold, 3.5)) / osuCurrent.StrainTime) **+ Math.Max(0, sdstrainmult)**;
```

Reply:
...How is your comment helping? You've pasted in a long formula without explanation. I'd like to know *specific details of why this formula was chosen*. In a *human-readable* and *human-understandable* manner.

Please understand that this code has **zero** chance of getting merged if nobody else will be able to understand it afterwards.